packages/@aws-cdk/aws-route53-targets/test/integ.api-gatewayv2-domain-name.ts

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env node
+import * as apigwv2 from '@aws-cdk/aws-apigatewayv2';
+import * as acm from '@aws-cdk/aws-certificatemanager';
+import * as route53 from '@aws-cdk/aws-route53';
+import { App, Stack } from '@aws-cdk/core';
+import { Construct } from 'constructs';
+import * as targets from '../lib';
+
+class TestStack extends Stack {
+  constructor(scope: Construct, id: string) {
+    super(scope, id);
+
+    const domainName = 'example.com';
+    const certArn = 'arn:aws:acm:us-east-1:111111111111:certificate';
+
+    const certificate = acm.Certificate.fromCertificateArn(this, 'cert', certArn);
```

Comment:
integration tests should be deployable (i.e. running `npm integ *your test.js*` will deploy the stack and automatically write the `.expected` file)

should this test be creating a certificate and then passing it's ARN? I can't imagine it deploys with the hardcoded `certArn` value